app.py

```diff
@@ -1,36 +1,151 @@
-from flask import Flask
-import requests
-import json
+from flask import Flask, request
+import requests, json, pymongo
+from pymongo import MongoClient
+
+def connect():
+    connection = MongoClient("ds031257.mlab.com",31257)
+    handle = connection["general_info_database"]
+    handle.authenticate("admin","admin1")
+    return handle
+
+handle=connect()
+db = handle.general_info_database
 app = Flask(__name__)
-@app.route('/',methods = ['GET'])
-URL = 'http://api.culpa.info/courses/department_id/7'
-
-def getAllData():
-    return null
-def getProfessors():
-    r = requests.get(URL)
-    json_data = json.loads(r.text)
-    array = [len(json_data['courses'])]
-    for i in range(len(json_data['courses'])):
-        array[i] = str(json_data['courses'][i]['name'])
-    return array
-
-def getCourseNames():
-    r = requests.get(URL)
-    jason_data = json.loads(r.text)
-    array = ""
-    for i in range(len(jason_data['courses'])):
-        array = array + str(jason_data['courses'][i]['name'])+'x'
-    return array
+URL = 'doc.json'
+db_users = handle.users_database
+
+
+class Course:
```

Comment:
Why do we need this class? If it's just a `struct ` of data, can we use a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple)? If it's supposed to interface with Mongo, can we use a 3rd-party Mongo library to automate the conversion from Mongo BSON to Python object?